File: QTD.SPA/src/app/_DtoModels/ILA_Topic/ILA_Topic.ts
import { ILA_Topic_Link } from '@models/ILA_Topic_Link/ILA_Topic_Link';
import { Entity } from '../Entity';

export class ILA_Topic extends Entity {
  name!: string;
  isPriority!: boolean;
  ilA_Topic_Links!: ILA_Topic_Link[];
}

File: QTD2.Domain/Entities/Core/ILA_Topic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using QTD2.Domain.Entities.Common;

namespace QTD2.Domain.Entities.Core
{
    public class ILA_Topic : Entity
    {
        public bool IsPriority { get; set; }

        public string Name { get; set; }

        public virtual ICollection<ILA_Topic_Link> ILA_Topic_Links { get; set; } = new List<ILA_Topic_Link>();

        public ILA_Topic(bool isPriority, string name)
        {
            IsPriority = isPriority;
            Name = name;
        }

        public ILA_Topic()
        {
        }
    }
}
